### ISSUE TITLE
allow for object groups. 

### ISSUE THREAD

**Issue by fabulousduck** [tsoding/nothing#437]
Object groups make it easy to duplicate complex objects consisting of multiple objects. Object groups in SVG terms are just a lot of rects with a wrapper around them

**Comment by rexim** [comment]
@fabulousduck groups work for me after #423 Consider this svg: [groups.svg.txt](https://github.com/tsoding/nothing/files/2476883/groups.svg.txt)

Levels compiles correctly. Game runs the game as expected. What exactly do you expect from groups?

**Comment by fabulousduck** [comment]
it seems that this is only a bug for rects that have not had their names changed. This happens for me only when i init a new rect, group it with another.

**Comment by rexim** [comment]
@fabulousduck could you please provide the bugged svg?

**Comment by rexim** [comment]
@fabulousduck your problems might be caused by the transformations applied on the rects. Search for `transform="scale(-1,1)"` in the svg text. Inkscape likes to insert them when you manipulate several objects at once. Unfortunately we don't support that. I'd recommend to avoid selecting several objects and trying to do transformations on them for now.

**Comment by rexim** [comment]
It would be nice to have a tool that sort of "normalizes" svg by applying all of the transformation leaving rects with absolute coordinates, but I don't know any of such tools.

**Comment by rexim** [comment]
@fabulousduck you may find this useful https://stackoverflow.com/questions/13329125/removing-transforms-in-svg-files Maybe we should even document that.

**Comment by fabulousduck** [comment]
I could probably whip up a small regex that does that next weekend when i have time for it. doesn't sound like the hardest problem in the world

**Comment by TheKnarf** [comment]
> It would be nice to have a tool that sort of "normalizes" svg by applying all of the transformation leaving rects with absolute coordinates, but I don't know any of such tools.

If I don't remember wrong there exists a few JavaScript based ones. [Svgo](https://github.com/svg/svgo) is the one that comes to mind, but I think there might be others as well.

**Comment by rexim** [comment]
Superseded by #592
